README.md

```diff
@@ -1,21 +1,72 @@
 # EngFlow APIs
 
-This repository contains the gRPC service definitions for public EngFlow APIs.
+This repository contains the gRPC service definitions for public EngFlow APIs:
+
+- **authentication API**
+- **notification queue API**
+- **eventstore API**.
 
 This repository intentionally does not provide generated code for any languages
-to keep it free of clutter.
+to keep it free of clutter. In other words,  it only provides
+[protocol buffers](https://developers.google.com/protocol-buffers) definitions.
 
 
 ## Building
 
-Use [Bazel](https://bazel.build/) >= 5.x.
+Use [Bazel](https://bazel.build/) 5.x (or higher) to build the library.
+Older versions of Bazel may work as well, but are not officially supported.
+If you do not have Bazel installed already, we recommend installing
+[bazelisk](https://github.com/bazelbuild/bazelisk),
+which automatically fetches the latest stable version of Bazel by default.
+
+To build all libraries run:
 
-> Note: older versions of Bazel may work as well, but are not officially supported.
+```bash
+bazel build //...
+```
 
-First, [install bazel](https://docs.bazel.build/versions/master/install.html).
+## Using the APIs in your Bazel project
 
-To build all libraries:
+In order to integrate the `engflowapis` into your Bazel project, you should
+include it using `http_archive`.
+
+We recommend to download a specific commit from GitHub and then store and reference
+it in any storage service. To do so, you may download a repository `zip` file using a
+commit sha:
 
 ```
```

Comment:
Please add syntax hint to every opening code fence?

Reply:
Done. Bazel is not yet supported by the markdown language, right?

Reply:
I've mostly seen `bzl` used as the syntax hint. Not sure if that's actually meaningful to GitHub's parser though.

Reply:
https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L6253-L6272